### ISSUE TITLE
Extended CustomTitleBar Bug

### ISSUE THREAD

**Issue by HesamKashefi** [Windows-XAML/Template10#1459]
Hi, When the CustomTitleBar is set to Extended, The upper segment of the HambergurMenu Button which is now under the hided title bar is not clickable ! only a portion of the button from it's bottom is clickable.

**Comment by HesamKashefi** [comment]
one more thing! 
It just turned out that the Back Button will also appear on top of the Hamburger Button and 
the 3 Buttons of Exit ,Minimize and Maximize will be on top of whatever is in the right side of Page Header for example a CommandBar!

**Comment by JerryNixon** [comment]
This is a problem. For sure. Need to create a built-in margin when it is visible. To be honest, this is the developer's problem. And prob not correct for Template 10 to build-in. What do you think?

**Comment by HesamKashefi** [comment]
I was just struggling with this right now, there are strange behaviors in the Title bar, I don't know what's wrong with it but when I added this style 
`<common:BootStrapper x:Class="TestingCustomTitleBar.App"
                     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:common="using:Template10.Common"
                     xmlns:controls="using:Template10.Controls">

    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="Styles\Custom.xaml" />
            </ResourceDictionary.MergedDictionaries>

            

            <Style TargetType="controls:CustomTitleBar">
                <Setter Property="Extended" Value="True" />
            </Style>
        </ResourceDictionary>
    </Application.Resources>

</common:BootStrapper>`

 it normally extends it but after it, It doesn't matter whether I set it to False or I completely remove the style it will remains extended for ever! even if I clean , remove bin folder and obj folder and uninstall the app and Rebuild the solution!! ( also I set the property Copy always for App.xaml.cs but nothing!)
I even used this method
`CustomTitleBar.Setup();`
but nothing !
I think I have to reinstall Windows! 😒😁
I took a look at the source code and I think it doesn't have any problem leading this!

Anyway let's talk about the main problem!, in my opinion there are two ways : 

1. The Extended property remains so there must be a margin (useless) or some kind of trick with ZIndex? (needs to set a custom layout in Title bar as the UWP Samlpe does)

2. Instead of the Extended property there may be a Extended Content by which we could set even controls in the Title Bar again as the UWP Sample does
